### PR TITLE
code style

### PR DESCRIPTION
**Pull request by jrmartino**
apply duraspace code style
change parent pom from spring boot to eclipse-pass/main

code style has been applied and verifies. There are still some issues with integration tests failing to run.

### CODE REVIEW

**Review by jabrah — APPROVED** [review]
Code style changes look reasonable. My maven build fails building the Docker image, but that's only used for ITs, which we'll address in a later ticket

File: pom.xml [hunk 1]
```diff
@@ -2,393 +2,419 @@
...
+  </properties>
 
-    <build>
-        <finalName>pass-doi-service</finalName>
-        <plugins>
-            <plugin>
-                <groupId>org.springframework.boot</groupId>
-                <artifactId>spring-boot-maven-plugin</artifactId>
-            </plugin>
+  <dependencyManagement>
```
Comment (bbranan) [inline pom.xml:99]: I'd suggest adding a comment here to call out the purpose of this section (as an alternative to using the Spring boot parent pom). Otherwise, it'll be easy to forget why this change was made.

**Review by bbranan — APPROVED** [review]
